mixin/helpers.go

```diff
@@ -116,13 +118,41 @@ func setUpMixDir(bundle string, upstreamVer, mixVer int) error {
 	return ioutil.WriteFile(filepath.Join(mixWS, "upstreamversion"), []byte(fmt.Sprintf("%d", upstreamVer)), 0644)
 }
 
-func setUpMixDirIfNeeded(bundle string, ver, mixVer int) error {
+func setUpMixDirIfNeeded(ver, mixVer int) error {
 	var err error
 	if _, err = os.Stat(filepath.Join(mixWS, "builder.conf")); os.IsNotExist(err) {
-		err = setUpMixDir(bundle, ver, mixVer)
+		err = setUpMixDir(ver, mixVer)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
+
+func parseHeaderNoopInstall(pkg, installOut string) (string, error) {
+	parts := strings.Split(installOut, "Installing:\n")
+	if len(parts) < 2 {
+		// dnf failure - no such package
+		return "", errors.New("no such package")
+	}
+	var r = regexp.MustCompile(fmt.Sprintf(` (%s)\s+\S+\s+\S+\s+(\S+)\s+\S+ \S+\n`, pkg))
+	matches := r.FindStringSubmatch(parts[1])
+	if len(matches) == 3 {
+		return matches[2], nil
+	}
+	return "", errors.New("unable to find repo for package")
+}
+
+func getPackageRepo(pkg string, ver int, config string) (string, error) {
+	packagerCmd := []string{
+		"dnf",
+		"--config=" + config,
+		fmt.Sprintf("--releasever=%d", ver),
+		"install",
+		"--assumeno",
+		pkg,
+	}
+
+	outBuf, _ := helpers.RunCommandOutput(packagerCmd[0], packagerCmd[1:]...)
```

Comment:
Would be useful for later people to have a comment saying we ignore this error because --assumeno makes the command always return error.